src/cool_seq_tool/resources/status.py

```diff
@@ -120,9 +120,12 @@ async def check_status(
     else:
         status["liftover"] = True
 
-    sanitized_url = redact_auth_from_url(UTA_DB_URL)
     try:
+        parsed_result = ParseResult(urlparse(db_url))
+        sanitized_url = parsed_result.sanitized_url
```

Comment:
Also, this line has to be outside the `try`, otherwise it's not in-scope for the except statements.